### ISSUE TITLE
Legal living Dex scripts have broken nicknames

### ISSUE THREAD

**Issue by rhyspiece** [FlagBrew/PKSM-Scripts#28]
Please see attached image, I'm running the latest 'legal living dex' script and quite a few Pokémon have blank names or names that display as タ I think? (Ivysaur, Venusaur, Charmeleon to name a few).
![15967052982774724234312959874979](https://user-images.githubusercontent.com/69300515/89514408-b9946880-d7cd-11ea-8f0a-a552eb8dc5b8.jpg)

**Comment by GriffinG1** [comment]
That’s Japanese, not a “broken nickname”.

**Comment by rhyspiece** [comment]
> That’s Japanese, not a “broken nickname”.

Yeah I'm well aware of that, but when I view the Pokémon in game, it has no nickname at all.
